app/Models/Offices.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Offices extends Model
```

Comment:
```suggestion
class Office extends Model
```

Đổi cả tên file giúp mình. Xem thêm nhé https://webdevetc.com/blog/laravel-naming-conventions/#section_naming-models-in-laravel

Reply:
> > Để cập nhật một thuộc tính nằm trong blacklist, ta tiến hành tạo đối tượng cho thuộc tính và set giá trị cho nó sau đó sử dụng hàm save() để lưu lại.
> 
> cho mình 1 đoạn code ví dụ nha

![image](https://user-images.githubusercontent.com/109664408/211618773-c4bbace0-9604-4e07-a7e3-4bc250a1a458.png)